portfolio/src/main/webapp/charts-script.js

```diff
@@ -0,0 +1,71 @@
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawCharts);
+
+/* drawCharts draws all of the charts to display. It is called in setOnLoadCallback. 
+The reasoning behind drawCharts is that setOnLoadCallback executes one function on load,
+ and drawCharts provides a centralized place for which to load all of the charts on the website.
+*/
+function drawCharts(){
+    //Since we draw charts from different pages, we only want to add in particular charts if they are defined on that page.
+    var pieChart = document.getElementById('pie-chart-container');
+    if (typeof(pieChart) != 'undefined' && pieChart != null){
+        drawPieChart();
+    }
+
+    var gwChart = document.getElementById('globalwarming-chart-container');
+    if (typeof(gwChart) != 'undefined' && gwChart != null) {
+        drawGlobalWarmingChart();
+    }
+
+}
+
+/** Creates a chart and adds it to the page. */
+function drawPieChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('string', 'Pet Species');
+  data.addColumn('number', 'Count');
+        data.addRows([
+          ['Cats', 2],
+          ['Dogs', 2],
+          ['Reptiles', 4],
+          ['Birds', 3]
+        ]);
+
+  const options = {
+    'title': 'My Pets',
+    'width':500,
+    'height':400
+  };
+
+  const chart = new google.visualization.PieChart(
+      document.getElementById('pie-chart-container'));
+
+  chart.draw(data, options);
+
+}
+
+/** Fetches bigfoot sightings data and uses it to create a chart. */
+function drawGlobalWarmingChart() {
+  fetch('/global-warming-sentiment-data').then(response => response.json())
+  .then((globalWarmingSentiment) => {
+    const data = new google.visualization.DataTable();
+    //have Sentiment and frequency/amount of occurrence
+    data.addColumn('string', 'Tweet');
+    data.addColumn('number', 'Sentiment');
+    Object.keys(globalWarmingSentiment).forEach((tweet) => {
+      data.addRow([tweet, globalWarmingSentiment[tweet]]);
+    });
+
+    const options = {
+      'title': 'Global Warming Sentiment',
+      'width':600,
+      'height':500,
+       histogram: { bucketSize: 1 }
+    };
+
+    const chart = new google.visualization.Histogram(
+        document.getElementById('globalwarming-chart-container'));
```

Comment:
I'm wondering if it's easy to:
1. For horizontal axis, show "Yes"/"No" rather than 0/1 (and 2 makes it confusing)?
2. For vertical axis, add unit so that users know what 300 means?